MissionScanner.cpp

```diff
@@ -87,10 +109,12 @@ void OutputHelp()
 	std::cout << "Review the publically exported infromation contained in Outpost 2 mission DLLs" << std::endl;
 	std::cout << std::endl;
 	std::cout << "+++ COMMANDS +++" << std::endl;
-	std::cout << "  * MissionScanner (archivename.(vol|clm) | directory)..." << std::endl;
+	std::cout << "  * MissionScanner (archivename.(vol|clm) | directory)... [-L]" << std::endl;
 	std::cout << std::endl;
 	std::cout << "+++ OPTIONAL ARGUMENTS +++" << std::endl;
 	std::cout << "  -H / --Help / -?: Displays help information." << std::endl;
+	std::cout << "  -L / --Legend: Remove legend." << std::endl;
```

Comment:
This seems a bit backwards in a way. Typically options are used to enable stuff, rather than disable stuff. When disabling something with a long option, most Linux utilities will use a name such as `--no-thingToTurnOff`. Maybe we should use `--no-legend`? Though I'm not sure what to suggest for the short option name `-L`.

Reply:
It would read nicer and I think more logically if it just turned the legend on when the switch was provided.